pkg/image/oci/registry_provider.go

```diff
@@ -38,7 +38,12 @@ func (p *RegistryImageProvider) Provide() (*image.Image, error) {
 		return nil, err
 	}
 
-	ref, err := name.ParseReference(p.imageStr)
+	options := make([]name.Option, 0, 2)
+	if p.registryOptions.InsecureSkipTLSVerify {
```

Comment:
I was looking in to: https://github.com/anchore/grype/issues/334 and came across this PR.

This change will cause the [registry to be accessed using `http://`](https://github.com/google/go-containerregistry/blob/7e30746dde2f5a19b609f885dfef4e7b1d1503ad/pkg/name/registry.go#L76), which is not the same thing as [skipping TLS verification](https://github.com/anchore/stereoscope/blob/d577f30b19e64b4c4b239bd2f23403014fa612fd/pkg/image/oci/registry_provider.go#L74) I think a more appropriate change would be to add another option specific for accessing the registry using `http://`.